swarm.go

```diff
@@ -497,7 +497,7 @@ func (s *Swarm) Stop() error {
 	}
 
 	if s.tags != nil {
-		err := s.stateStore.Put("tags", s.tags)
+		err := s.tags.Persist()
```

Comment:
as we discussed this is not sufficient for correct tag persistance when crashes happen.